### DISCUSSION TITLE
have `man` automatically open in new tab?

### DISCUSSION THREAD

**Discussion by chrisgrieser** [wezterm/wezterm#3420] [Q&A] [answered]
Hi, new to WezTerm and loving it.

I was wondering whether it is possible to have `man …` automatically open in a new tab? 

I assume it would require adding a custom function my shell's rcfile, but I am not sure what I need to make the shell tell wezterm to perform something other than the wezterm cli (which apparently does not include a "open as new tab in existing instance option)

**Comment by wez** [comment, marked as answer]
You could try something like:

```
alias man='wezterm cli spawn -- man'
```

**Reply by chrisgrieser** [reply]
oh okay, that was easier than expected. Works like a charm, thanks you 🥳
